wall_e/src/resources/cogs/reminders.py

```diff
@@ -105,8 +128,9 @@ async def remindmein(self, ctx, *args):
             if e_obj is not False:
                 await ctx.send(embed=e_obj)
             return
+        time_struct = time_struct.utctimetuple()  # Server runs in UTC, so time has to be converted
         expire_seconds = int(mktime(time_struct) - time.time())
-        dt = datetime.datetime.now()
+        dt = datetime.datetime.now(tz=pytz.utc)  # Note that datetime.utcnow() returns naive datetime
         b = dt + datetime.timedelta(seconds=expire_seconds)  # days, seconds, then other fields.
```

Comment:
please change this to a better variable name then just `b`

Reply:
That was there in the original code so I didn't feel fit to change it (this was meant to be a feature addition rather than refactoring), but can do

Reply:
yea, poor coding practice on our end. just taking advantage of your PR to fix it now :stuck_out_tongue: